### DISCUSSION TITLE
Adding files to the build

### DISCUSSION THREAD

**Discussion by ExcelSup** [DigitalLogicSimCommunity/Digital-Logic-Sim-CE#81] [Q&A]
For context, I have been using this program for a while but came to a stand still when I wanted to use the 7 segment displays which are present in the original but not in this version. I tried first by myself manually adding files that DHMike57 made to the source code that I downloaded and then used unity to build my changed source code. In general, is there any way to actually  do this?

**Comment by phillipfu96** [comment]
i have the same issue, how can i get it
